spring-cloud-starter-stream-sink-cassandra/src/main/java/org/springframework/cloud/stream/app/cassandra/util/NonValidatingSSLContextFactory.java

```diff
@@ -0,0 +1,32 @@
+package org.springframework.cloud.stream.app.cassandra.util;
+
+import javax.net.ssl.*;
+import java.security.GeneralSecurityException;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+
+public class NonValidatingSSLContextFactory {
+    public static SSLContext getSslContext() {
+
+        TrustManager[] trustAllCerts = new TrustManager[] { new X509TrustManager() {
+            public X509Certificate[] getAcceptedIssuers() {
+                return new X509Certificate[0];
+            }
+
+            public void checkClientTrusted(java.security.cert.X509Certificate[] certs, String authType) {
+            }
+
+            public void checkServerTrusted(java.security.cert.X509Certificate[] certs, String authType) {
+            }
+        } };
+
+        SSLContext sc = null;
+        try {
+            sc = SSLContext.getInstance("SSL");
+            sc.init(null, trustAllCerts, new SecureRandom());
+        } catch (GeneralSecurityException e) {
+        }
```

Comment:
That's not good to swallow exception. And I think it even would be better to pull all the exception to the `throws` of the `getSslContext()` and let the user to deal with the exception already.